x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/expanded_row.tsx

```diff
@@ -157,8 +158,14 @@ export const ExpandedRow: FC<Props> = ({ item }) => {
     title: i18n.translate('xpack.ml.dataframe.analyticsList.expandedRow.tabs.jobSettings.state', {
       defaultMessage: 'State',
     }),
-    items: Object.entries(stateValues).map(s => {
-      return { title: s[0].toString(), description: getItemDescription(s[1]) };
+    items: Object.entries(stateValues).map(([stateKey, stateValue]) => {
+      if (stateKey.toString() === 'state') {
```

Comment:
complete nit, `stateKey.toString()` is called   3 times in here. it could be pulled out into a `title` var at the top of the function.

Reply:
Done in https://github.com/elastic/kibana/pull/60249/commits/d3df262ad58866dd612cd3621502d911329d582c